app/models/pusher.rb

```diff
@@ -60,7 +61,14 @@ def find
 
     @rubygem = Rubygem.name_is(name).first || Rubygem.new(name: name)
 
-    unless @rubygem.new_record?
+    if @rubygem.new_record?
+      Rubygem.downloaded(50).each do |rubygem|
+        if Gem::Text.levenstein_distance(name.downcase, rubygem.name.downcase) < 3
```

Comment:
Pretty sure it's `levenshtein_distance`? Unrelated, if it's fast enough I'd like to do the top 100 (or top 1000?). Maybe `Rubygem.downloded(1000).pluck(:name).first{|n| Gem::Text.levenshtein_distance(name.downcase, n.downcase) < 3 }`?